apps/user-office-backend/src/eventHandlers/messageBroker.ts

```diff
@@ -90,9 +95,21 @@ const getProposalMessageData = async (proposal: Proposal) => {
     proposal.primaryKey
   );
 
+  const maybeInstrument = await instrumentDataSource.getInstrumentByProposalPk(
+    proposal.primaryKey
+  );
+
+  const instrument = maybeInstrument
+    ? {
+        id: maybeInstrument.id,
+        shortCode: maybeInstrument.shortCode,
+      }
+    : undefined;
```

Comment:
If we don't have an instrument do we need it to be `undefined` or we can skip it from the message Data. But eventually It will all be about its usage on the receiving consumer.

Reply:
I tried to stick to the convention in this file. I could see that `newStatus` if not set would be undefined.

Reply:
Okay can see the point.